routes/index.js

```diff
@@ -1,31 +1,66 @@
 const apiRouter = require('express').Router();
-const { getAllProducts, getProductById } = require('../db');
+const jwt = require('jsonwebtoken')
+const JWT_SECRET = 'codebrigaderules'
```

Comment:
Use an environment variable for this instead of hardcoding.
```js
require('dotenv').config();
const JWT_SECRET =  process.env.JWT_SECRET || 'codebrigaderules';
```